test/measure.service.test.js

```diff
@@ -0,0 +1,127 @@
+require('../src/util/dbconfig');
+const testMeasure = require('./fixtures/testMeasure.json');
+const testLibrary = require('./fixtures/testLibrary.json');
+const testPatient = require('./fixtures/testPatient.json');
+const { testSetup, cleanUpDb } = require('./populateTestData');
+const supertest = require('supertest');
+const { buildConfig } = require('../src/util/config');
+const { initialize } = require('../src/server/server');
+const config = buildConfig();
+const server = initialize(config);
+
+
+const postRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+};
+const putRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'PUT',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+};
+const getRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'GET',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+}
+
+const deleteRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'DELETE',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+}
+describe('measure.service', () => {
+  beforeAll(async () => {
+    await testSetup(testMeasure, testPatient, testLibrary);
+  });
+  afterAll(async () => {
+    await cleanUpDb();
+  });
+  test('test', async () => {
+    await supertest(server.app).get('/4_0_0/metadata').expect(200);
+  });
+  describe('create', () => {
+    test('test create with correct headers', async () => {
+      await supertest(server.app)
+        .post('/4_0_0/Measure')
+        .send(postRequest)
+        .expect(200)
+        .then(async response => {
+          // Check the response
+          expect(response.body._id != null);
```

Comment:
2 things

1) this checks for the presence of a mongo-generated `_id`, which isn't bad, but probably what makes more sense to do with a POST request is to make sure the server generated a new ID, and that the content location header of the response indicates where to find it. So we should be accessing response.body.id, and making sure it was modified to some uuid that isn't the ID given by the test fixture since that is the expected behavior. You may also want to assert something about the Content-Location header.

2) For `jest` assertions, there is a [convenience function](https://jestjs.io/docs/expect#tobenull) for null assertions, and you can chain it with a `.not` for negation: `expect(something).not.toBeNull()`